File: docker/Dockerfile.swift
RUN curl -s -L https://download.swift.org/swift-5.10.1-release/ubuntu2404/swift-5.10.1-RELEASE/swift-5.10.1-RELEASE-ubuntu24.04.tar.gz | tar xvz --transform 's/^swift-5.10.1-RELEASE-ubuntu24.04//' -C /
RUN apt install -y g++ llvm-17
ADD apps/swift/usr /usr/
ADD apps/swift/opt /opt/
    
USER $USERNAME

File: docker/zig/Dockerfile.zig
ARG FROM_IMAGE=ryusatgat/ubuntu
FROM $FROM_IMAGE

RUN curl -s -L https://ziglang.org/builds/zig-linux-x86_64-0.14.0-dev.1550+4fba7336a.tar.xz | tar xvJ --transform 's/^zig-linux-x86_64-0.14.0-dev.1550+4fba7336a/zig/' -C /opt && ln -s /opt/zig/zig /usr/local/bin

USER $USERNAME
